src/main/java/appeng/parts/networking/PartQuartzFiber.java

```diff
@@ -131,14 +133,46 @@ public void onPlacement( final EntityPlayer player, final EnumHand hand, final I
 	}
 
 	@Override
-	public double extractAEPower( final double amt, final Actionable mode, final Set<IEnergyGrid> seen )
+	public Collection<IEnergyGridProvider> providers()
+	{
+		Collection<IEnergyGridProvider> stuff = new LinkedList<>();
+
+		try
+		{
+			final IEnergyGrid eg = this.getProxy().getEnergy();
+			stuff.add( eg );
+		}
+		catch( final GridAccessException e )
+		{
+			// :P
+		}
+
+		try
+		{
+			final IEnergyGrid eg = this.outerProxy.getEnergy();
+			stuff.add( eg );
+		}
+		catch( final GridAccessException e )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1166)